eve/workers/unit_and_feature_tests/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM spotify/kafka
+
+#
+# Install apt packages needed by backbeat and buildbot_worker
+#
+COPY ./backbeat_packages.list ./buildbot_worker_packages.list /tmp/
+
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash - \
+    && cat /tmp/*packages.list | xargs apt-get install -y \
+    && pip install pip==9.0.1 \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -f /tmp/packages.list
+
+#
+# Add user eve
+#
+RUN adduser -u 1042 --home /home/eve --disabled-password --gecos "" eve \
+    && adduser eve sudo \
+    && sed -ri 's/(%sudo.*)ALL$/\1NOPASSWD:ALL/' /etc/sudoers
+
+#
+# Run buildbot-worker on startup through supervisor
+#
+ARG BUILDBOT_VERSION=0.9.1
+
+RUN pip install buildbot-worker==$BUILDBOT_VERSION
+ADD supervisor/buildbot_worker.conf /etc/supervisor/conf.d/
+
+#
+# Configure build env
+#
+ENV CXX=g++-4.9
+
+#
+# Install nodejs packages needed by backbeat
+#
+COPY package.json /usr/lib/package.json
+WORKDIR /usr/lib/
```

Comment:
`/usr/lib` seems to be a rather uncommon root directory for our installs, what about `/opt` or even `~`?

Reply:
Or rather, `/opt/backbeat` or `~/backbeat`

Reply:
We can change it to whichever is preferred

Reply:
We're also using `/usr/src/app` for zenko deployments.

Reply:
@rahulreddy @RachedBenMustapha what would you advise? I was just surprised to see `/usr/lib` but I'm not sure where it should go instead.

Reply:
g

Well, we install the dependencies in /usr/lib so they are available globally on the worker.
Because at that point (worker creation) we don't yet have the backbeat repo cloned so we can't install them locally in the repo files.
This way we don't have to bother about installing dependecies or changing directory before runnig our npm commands in the ci (eve/main.yml).